spec/features/collection_drafts/publish_collection_draft_spec.rb

```diff
@@ -3,6 +3,32 @@
     login
   end
 
+  context 'when publishing a collection draft with newly added forms, values, features when UMM-C is updated' do
+    before do
+      @email_count = ActionMailer::Base.deliveries.count
+
+      draft = create(:full_collection_draft, user: User.where(urs_uid: 'testuser').first, draft_short_name: '12345', draft_entry_title: 'Draft Title')
```

Comment:
I would generally avoid using specific short_names/titles when publishing unless you need to.  Especially if they do not contain a random component.

Reply:
ok